build/references/coreth-arc20s.md

```diff
@@ -0,0 +1,108 @@
+# Coreth ANTs and ARC20s
+
+## ANTs on the C Chain
+
+### Why move an ANT from the X Chain to the C Chain?
```

Comment:
As a convention should we use dashes when talking about chains?

> the X Chain to the C Chain.
or

> the X-Chain to the C-Chain.

Reply:
Updated